gradio/events.py

```diff
@@ -199,10 +208,13 @@ def clear(
         inputs: List[Component],
         outputs: List[Component],
         api_name: AnyStr = None,
+        status_tracker: Optional[StatusTracker] = None,
+        scroll_to_output: bool = False,
```

Comment:
Was there a reason these parameters were omitted originally or was this just simply an oversight on our part?

Reply:
I think an oversight. @aliabid94 do you know?